CHANGES.md

```diff
@@ -3,6 +3,10 @@
 This document describes the relevant changes between releases of the OCM API
 SDK.
 
+## 0.1.436
+- Add state struct to node pool
```

Comment:
Take a look at the convention for bumping api model: https://github.com/openshift-online/ocm-sdk-go/pull/986/files#diff-d975bf659606195d2165918f93e1cf680ef68ea3c9cab994f033705fea8238b2R16